packages/admin/admin-color-picker/src/FinalFormColorPicker.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from "react";
+import { FieldRenderProps } from "react-final-form";
+
+import { ColorPicker, ColorPickerProps } from "./ColorPicker";
+
+export type FinalFormColorPickerProps = ColorPickerProps & FieldRenderProps<string, HTMLInputElement | HTMLTextAreaElement>;
```

Comment:
TextArea?

Reply:
@nsams The `InputBase` (used by `InputWithPropper`, which is used by all pickers) has all types for Input & TextArea, because you could technically pass in a TextArea instead of the default Input. For the FinalForm-Pickers to be compatible with the types, the TextArea type is needed here. 

To remove the TextArea types we would have to override all the event-types of `InputBaseProps` and maybe prevent the ability to pass in a TextArea for the input-component prop. I don't think this is worth it. What do you think?

Reply:
I understand.

If you use `export type FinalFormColorPickerProps = ColorPickerProps & InputBaseProps` I would not even have noticed that. Can you change that?

Reply:
@nsams but I need to use the `FieldRenderProps` from `react-final-form` to use this inside a final-form, right? 🤔

Reply:
@nsams do you think this is an issue, or can we keep it the way it is?

Currently (by default of MuiInputBase), you just have the additional ability to use a TextField instead of an Input. It's probably not a feature we need, but I don't think this causes any issues.